webview.h

```diff
@@ -1048,12 +1074,29 @@ class edge_chromium : public browser {
       }
       return S_OK;
     }
+    HRESULT STDMETHODCALLTYPE
+    Invoke(ICoreWebView2 *sender,
+           ICoreWebView2NavigationCompletedEventArgs *args) {
+      PWSTR uri;
+      sender->get_Source(&uri);
+      std::wstring_convert<std::codecvt_utf8<wchar_t>, wchar_t> converter;
```

Comment:
Do not use uses deprecated string conversion functions, you will receive compiler warnings. Use winrt::to_string()